geth/api/api.go

```diff
@@ -146,11 +146,13 @@ func (api *StatusAPI) VerifyAccountPassword(keyStoreDir, address, password strin
 // using provided password. Once verification is done, decrypted key is injected into Whisper (as a single identity,
 // all previous identities are removed).
 func (api *StatusAPI) SelectAccount(address, password string) error {
+	api.b.jailManager.Stop()
```

Comment:
Hm. Currently `JailManager.Stop` doesn't stop Jail, it just resets it to an initial state. Can you make a comment above the original method like `// FIXME(oleg-raev): This method doesn't make stop, it rather resets its cells to an initial state and should be properly renamed, for example: ResetCells`?